<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>978</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XMA Lab</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../calibrationtool_resources.qrc">
    <normaloff>:/images/resource-files/icons/XMALab-Icon.png</normaloff>:/images/resource-files/icons/XMALab-Icon.png</iconset>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="margin">
     <number>6</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="mainframe">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="QFrame" name="imageMainFrame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4"/>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QFrame" name="startMainFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButtonLoad_Project">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Load .xma file</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonNew_Project">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>New dataset</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QScrollArea" name="imageScrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="imageScrollAreaContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>826</width>
            <height>69</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="startTrialFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer_12">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButtonNewTrial">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Add trial</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="labelCalibrateFirst">
         <property name="text">
          <string>You first have to calibrate your cameras!</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionExport3D_Points"/>
     <addaction name="actionExport2D_Points"/>
     <addaction name="actionRigidBodyTransformations"/>
     <addaction name="actionMarkertoMarkerDistances"/>
     <addaction name="separator"/>
     <addaction name="actionExport_Undistorted_Trial_images_for_Maya"/>
     <addaction name="actionMayaCams_2_0"/>
     <addaction name="actionMayaCams"/>
     <addaction name="separator"/>
     <addaction name="actionExportAll"/>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionImport2D_Points"/>
     <addaction name="actionImportTrial"/>
    </widget>
    <addaction name="actionNew_Project"/>
    <addaction name="actionLoad_Project"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_Project_as"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="menuImport"/>
    <addaction name="separator"/>
    <addaction name="actionUndistort_sequence"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionAbout"/>
    <addaction name="actionConsole"/>
    <addaction name="actionDetailed_View"/>
    <addaction name="actionPlot"/>
    <addaction name="action3D_world_view"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAddCamera">
   <property name="text">
    <string>Add camera</string>
   </property>
  </action>
  <action name="actionLoadGrid">
   <property name="text">
    <string>Load calibration target</string>
   </property>
  </action>
  <action name="actionRefineAllCams">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Optimize calibration target and cameras</string>
   </property>
  </action>
  <action name="actionCalibrate_Cam_2">
   <property name="text">
    <string>Calibrate Cam 2</string>
   </property>
  </action>
  <action name="actionShow_Detector_Options">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Detector Options</string>
   </property>
  </action>
  <action name="actionFit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit</string>
   </property>
  </action>
  <action name="action25">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>25%</string>
   </property>
  </action>
  <action name="action50">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
  </action>
  <action name="action100">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </action>
  <action name="action200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>200%</string>
   </property>
  </action>
  <action name="action400">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>400%</string>
   </property>
  </action>
  <action name="actionFreeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>free view</string>
   </property>
  </action>
  <action name="actionCameraView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>camera view</string>
   </property>
  </action>
  <action name="actionCam_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cam 2</string>
   </property>
  </action>
  <action name="actionTabbed_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tabbed View</string>
   </property>
  </action>
  <action name="actionTiled_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tiled View</string>
   </property>
  </action>
  <action name="actionFullCalibrate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>by computting camera and pose (selected frame)</string>
   </property>
  </action>
  <action name="actionPoseCalibrate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>by computing pose with calibrated camera data (selected frame)</string>
   </property>
  </action>
  <action name="actionInit_correspondances_from_other_Cameras">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>by computing pose with calibrated cameras and pose data (all frames/cameras)</string>
   </property>
  </action>
  <action name="actionSaveCalibModel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Calibrated Camera Parameters (Model based)</string>
   </property>
  </action>
  <action name="actionSaveProjectionMatrix">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Merged Projection matrix (DLT)</string>
   </property>
  </action>
  <action name="actionSaveCalibModelSingle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Calibrated Camera Parameters (Model with single Focus)</string>
   </property>
  </action>
  <action name="actionSaveAllParameters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save All Parameters</string>
   </property>
  </action>
  <action name="actionSaveCalibModelDLT">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Calibrated Camera Parameters (DLT)</string>
   </property>
  </action>
  <action name="actionDetected_blobs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>detected blobs</string>
   </property>
  </action>
  <action name="actionProjected_calibration_object">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>reference calibration points</string>
   </property>
  </action>
  <action name="actionDetected_calibration_points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>detected calibration points</string>
   </property>
  </action>
  <action name="actionKeypoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>keypoints</string>
   </property>
  </action>
  <action name="actionRecalibrate_based_on_Inlier">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Camera and pose (selected frame)</string>
   </property>
  </action>
  <action name="actionDetect_new_correspondances">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>detect new correspondances</string>
   </property>
  </action>
  <action name="actionShow_3D_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 3D View</string>
   </property>
  </action>
  <action name="actionCalibrate_Camera_all_frames">
   <property name="text">
    <string>calibrate Camera (all frames)</string>
   </property>
  </action>
  <action name="actionCalibrateCameraAllFrames">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Camera and pose (all frames)</string>
   </property>
  </action>
  <action name="actionSet_new_correspondances_based_on_Error">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>set Inlier based on Error</string>
   </property>
  </action>
  <action name="actionPose_selected_Frame">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pose (selected Frame)</string>
   </property>
  </action>
  <action name="actionSave_calibration_target">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save calibration target</string>
   </property>
  </action>
  <action name="actionSave_Merged_Projection_matrix_Model_Based">
   <property name="text">
    <string>Save Merged Projection matrix (Model)</string>
   </property>
  </action>
  <action name="actionSave_Merged_Projection_Matrix_Model_with_single_Focus">
   <property name="text">
    <string>Save Merged Projection Matrix (Model with single Focus)</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New dataset</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Load dataset</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save dataset</string>
   </property>
  </action>
  <action name="actionSave_Project_as">
   <property name="text">
    <string>Save dataset as</string>
   </property>
  </action>
  <action name="actionExportDLT">
   <property name="text">
    <string>DLT</string>
   </property>
  </action>
  <action name="actionExportLUT">
   <property name="text">
    <string>Undistortion</string>
   </property>
  </action>
  <action name="actionExportMayacams">
   <property name="text">
    <string>Mayacams</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close dataset</string>
   </property>
  </action>
  <action name="action3D_world_view">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3D world view</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionUndistort_sequence">
   <property name="text">
    <string>Undistort sequence</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExportAll">
   <property name="text">
    <string>XRayProject Matlab Format</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actionDetailed_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Detailed View</string>
   </property>
  </action>
  <action name="actionExport3D_Points">
   <property name="text">
    <string>3D Points</string>
   </property>
  </action>
  <action name="actionExport2D_Points">
   <property name="text">
    <string>2D Points</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
  <action name="actionRigidBodyTransformations">
   <property name="text">
    <string>RigidBodyTransformations</string>
   </property>
  </action>
  <action name="actionFiltered_RigidBody_Transformations">
   <property name="text">
    <string>Filtered RigidBody Transformations</string>
   </property>
  </action>
  <action name="actionExport_Undistorted_Trial_images_for_Maya">
   <property name="text">
    <string>Undistorted Trial images</string>
   </property>
  </action>
  <action name="actionImport2D_Points">
   <property name="text">
    <string>2D Points</string>
   </property>
  </action>
  <action name="actionMayaCams">
   <property name="text">
    <string>MayaCams</string>
   </property>
  </action>
  <action name="actionImportTrial">
   <property name="text">
    <string>Trial from xma-file</string>
   </property>
  </action>
  <action name="actionMayaCams_2_0">
   <property name="text">
    <string>MayaCams 2.0</string>
   </property>
  </action>
  <action name="actionMarkertoMarkerDistances">
   <property name="text">
    <string>Marker to Marker Distances</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../calibrationtool_resources.qrc"/>
 </resources>
 <connections/>
</ui>
